engine/resource/wscript

```diff
@@ -65,7 +73,7 @@ def _start_server(self, local_ip, port, configname=None):
     Logs.info('Starting test server at http://%s:%d' % (local_ip, port))
     configfile = waf_dynamo.create_test_server_config(self, port=port, ip=local_ip, config_name=configname)
 
-    proc = subprocess.Popen(["python", "-m", "http.server", str(port)],
+    proc = subprocess.Popen(["python", "-m", "RangeHTTPServer", str(port)],
```

Comment:
I added this module to our install script, since the default http.server doesn't support ranged requests (!?), and I wanted it for the unit tests.